app/lib/ctc_dependent_eligibility.rb

```diff
@@ -0,0 +1,9 @@
+require "date"
+
+class CtcDependentEligibility
+  CUT_OFF_BIRTHDATE = Date.new(2004, 1, 1)
+
+  def self.eligible?(birthdate:)
```

Comment:
i believe we should add the qualifying relationship logic to this guy as well.

Reply:
Yeah for sure — still working but on the class that governs that